Bots/DotNet/EchoSkillBot-3.1/Properties/launchSettings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:35405/",
```

Comment:
Dont remember the full list of ports, just mentioning this to ensure is in the correct one

Reply:
I used the ports that the old netcore 2.1 bots were using.